web/src/features/fireBehaviourCalculator/fuelTypes.ts

```diff
@@ -6,7 +8,10 @@ export interface FBCFuelType {
   crown_base_height: number | undefined
 }
 export class FuelTypes {
-  static lookup(key: string): FBCFuelType {
+  static lookup(key: string | undefined): FBCFuelType | null {
+    if (isUndefined(key)) {
```

Comment:
Cool. It's always bothered me that there's no straight forward built in method for this!